service/switchboardapp/src/test/java/com/switchboard/app/DeviceControllerTest.java

```diff
@@ -0,0 +1,105 @@
+package com.switchboard.app;
+
+import com.switchboard.app.controller.DeviceController;
+import com.switchboard.app.dao.DeviceDaoImpl;
+import com.switchboard.app.domain.DeviceEntity;
+import com.switchboard.app.exceptions.DeviceAlreadyExistsException;
+import com.switchboard.app.exceptions.DeviceNotFoundException;
```

Comment:
`import com.switchboard.app.domain.DeviceEntity;` needs to change to `import com.switchboard.app.entity.DeviceEntity;`

`import com.switchboard.app.exceptions.DeviceAlreadyExistsException;` and `import com.switchboard.app.exceptions.DeviceNotFoundException;` needs to change to `com.switchboard.app.exceptions.ExceptionType;`

If not changed, we will get a compile error on merge.

Reply:
Has been updated